influxql/iterator.gen.go

```diff
@@ -2297,6 +2304,7 @@ func (itr *floatBooleanExprIterator) Next() (*BooleanPoint, error) {
 			if itr.points == nil {
 				continue
 			}
+
```

Comment:
You need to include something for when the types are not the same otherwise you're leaving a hole (for boolean statements mostly).